Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -3265,6 +3265,22 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
 
         return list;
     }
+    case CSSPropertyMasonryAutoFlow: {
+        auto list = CSSValueList::createSpaceSeparated();
+        // MasonryAutoFlow information is stored in an OpionSet that should always 
+        // hold 2 bits of information. It should contain both Pack/Next and
+        // DefiniteFirst/Ordered.
+        ASSERT((style.masonryAutoFlow().contains(MasonryAutoFlow::Pack) || style.masonryAutoFlow().contains(MasonryAutoFlow::Next)) && (style.masonryAutoFlow().contains(MasonryAutoFlow::DefiniteFirst) || style.masonryAutoFlow().contains(MasonryAutoFlow::Ordered)));
+
+        if (style.masonryAutoFlow().contains(MasonryAutoFlow::Next))
+            list->append(cssValuePool.createIdentifierValue(CSSValueNext));
+        else if (style.masonryAutoFlow().contains(MasonryAutoFlow::DefiniteFirst))
```

Comment:
Is definite-first supposed to mean packed? The spec was kind of unclear on this when I read it.

Reply:
The spec if a little weird with this. `definite-first` is the default state for the order unless `ordered` is specified. So if you do not specify `ordered,` the behavior is always `definite-first` regardless of whether it is `pack` or `next`

Reply:
Can we add a comment then 😀?

Reply:
Done!